bazel/repositories.bzl

```diff
@@ -233,7 +233,7 @@ def _go_deps(skip_targets):
 def _rust_deps():
     external_http_archive(
         "rules_rust",
-        patches = ["@envoy//bazel:rules_rust.patch"],
+        # patches = ["@envoy//bazel:rules_rust.patch"],
```

Comment:
Shall we remove this line?

Reply:
before landing for sure - it doesnt pass ci atm - it was another attempt at probs the same thing ive tried before

not sure the exact issue - the upstream comment is something about _not_ exporting `CrateInfo` unless ... - but then our patch seemed to restrict that further, and the error is that `CrateInfo` is not being exported

ill look further

Reply:
SG. Let me know when it's ready for review.
/wait